README.md

```diff
@@ -42,3 +44,100 @@ You can also implement a new computation type that is used to display the differ
     (defmethod compute ((x (eql :maximum)) (metric metric))
       (reduce-samples metric #'max))
 
+Benchmark Suites
+----------------
+
+Analogous to unit testing, *performance testing* or *benchmark testing* is a useful way to measure performance regressions in a code base. Trivial-Benchmark has **preliminary support** for defining and running benchmarks.
+
+First, a *benchmark package* needs to be defined, which is essentially a normal Common Lisp package with some additional bookkeeping.
+
+    (define-benchmark-package #:my-app-benchmarks
+      (:use #:my-app)
+      (:export #:run-benchmarks))
+
+The macro `define-benchmark-package` automatically uses the `COMMON-LISP` and the `TRIVIAL-BENCHMARKS` packages.
+
+After the package has been defined, one can define benchmarks like normal Lisp function definitions. One uses the `define-benchmark` macro for this. Within the `define-benchmark` macro, one uses `with-benchmark-sampling` around forms that one wishes to measure.
```

Comment:
Using "one" to address the reader is awkward and a bit too informal. Just use the "you can" like you already do further down.